### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by balbuf**
Add basic readme with deprecation notice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# bases
```
Comment (kbeck303) [inline README.md:1]: the other heading is capitalized, do you want this to be too?

Comment (balbuf) [reply README.md:1]: it's kind of stylized as lowercase but it doesn't really matter either way

Comment (kbeck303) [reply README.md:1]: 👍

**Comment by kbeck303** [comment]
bob_ross_no_mistakes


/approved

**Pull request merged**